session/manager.go

```diff
@@ -83,7 +83,10 @@ type Storage interface {
 }
 
 // BalanceTrackerFactory returns a new instance of balance tracker
-type BalanceTrackerFactory func(consumer, provider, issuer identity.Identity) (BalanceTracker, error)
+type BalanceTrackerFactory func(consumer, provider, issuer identity.Identity) (PaymentEngine, error)
+
+// PaymentEngineFactory creates a new instance of payment engine
```

Comment:
Minor suggestion. End each comment sentence with a dot at the end. Godoc page generates methods documentation based on these comments.

Reply:
makes sense, although we'll need to refactor the whole codebase for it. #399